wazo_sysconfd/exceptions.py

```diff
@@ -0,0 +1,11 @@
+# Copyright 2022 The Wazo Authors  (see the AUTHORS file)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+
+from http.server import BaseHTTPRequestHandler
+
+
+class HttpReqError(Exception):
+    def __init__(self, code: int, message: str = None):
+        self.code = code
+        self.message = message or BaseHTTPRequestHandler.responses[code][1]
```

Comment:
The dependency on `BaseHTTPRequestHandler` looks brittle to me: what if `code` is not a standard code? Furthermore, returning a standard message is not very useful.

Reply:
I took this line from the original HttpReError (https://github.com/wazo-platform/xivo-lib-python/blob/d0f4a907aa5d35385f1ae1d70f58f2f794ab91fd/xivo/http_json_server.py#L81)
At least I will not break something.
If we need to improve that, so I think we should create another PR, not linked to the FastAPI migration.
Let me know what you think about that.